lighter/utils/model.py

```diff
@@ -53,14 +56,59 @@ def replace_layer_with_identity(model: Module, layer_name: str) -> Module:
     return replace_layer_with(model, layer_name, Identity())
 
 
-def remove_last_layer_sequentially(model: Module()) -> Sequential:
-    """Removes the last layer of a network and returns it as an nn.Sequential model.
-    Useful when a network is to be used as a backbone of an SSL model.
+def adjust_prefix_and_load_state_dict(model: Module, ckpt_path: str, ckpt_to_model_prefix: Dict[str, str] = None) -> Module:
```

Comment:
Wanted to make this a separate PR, but I somehow ended up pushing it here. Guess I'll just leave it here